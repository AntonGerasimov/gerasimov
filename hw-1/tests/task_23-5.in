205 119 505.5
