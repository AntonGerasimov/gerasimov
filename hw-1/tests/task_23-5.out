205
505.500000
w
